src/en/koharu/src/eu/kanade/tachiyomi/extension/en/koharu/KoharuDto.kt

```diff
@@ -60,8 +60,16 @@ class Data(
 @Serializable
 class DataKey(
     val id: Int,
+    val size: Double = 0.0,
     val public_key: String,
-)
+) {
+    fun readableSize() = when {

```

Comment:
Use GiB, MiB, KiB


or if you want to keep using GB, MB, kB you should calculate that (multiple of 1000). Wouldn't recommend it, as those units are ambiguous.

Reply:
Hello, thank you for pointing that out. The website seems to convert Bytes to MiB but writes it as MB. 
I'll still use the MB and GB format as I think it's more commonly understood.